boa_engine/src/vm/opcode/delete/mod.rs

```diff
@@ -61,3 +61,78 @@ impl Operation for DeletePropertyByValue {
         Ok(ShouldExit::False)
     }
 }
+
+/// `DeleteName` implements the Opcode Operation for `Opcode::DeleteName`
+///
+/// Operation:
+///  - Deletes a property by value of an object
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
```

Comment:
Do we need to implement `PartialEq`, `Eq` and `Hash` here? What benefits does it have? I see we also have it elsewhere, but I find it odd, since we never construct one of these structures.

Reply:
I just copied it from other Ops. I think it would be worth to delete those, because they impact compilation times.